src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD = 6;
+    private static final int STORAGE_LIST_SIZE = 2;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();

```

Comment:
registrationService is not a constant. Lets initialize this variable in method with annotation BeforeAll